core/cas-server-core-webflow-api/src/main/java/org/apereo/cas/web/flow/resolver/impl/AbstractCasWebflowEventResolver.java

```diff
@@ -494,11 +494,10 @@ protected Service resolveServiceFromAuthenticationRequest(final RequestContext c
         try {
             final Credential credential = getCredentialFromContext(context);
             AuthenticationResultBuilder builder = WebUtils.getAuthenticationResultBuilder(context);
-
             LOGGER.debug("Handling authentication transaction for credential [{}]", credential);
             final Service service = WebUtils.getService(context);
+            builder.getInitialAuthentication().get().addAttribute("providerId", context.getActiveFlow().getId());
```

Comment:
Why this?

Reply:
No need, I will remove.  Something I think I was going to try and forgot to back it out.